views/challenge/create.php

```diff
@@ -131,3 +146,8 @@
 
 <?php ActiveForm::end(); ?>
 <?php ModalDialog::end() ?>
+<script>
+    $(document).ready(function () {
+        document.getElementById("challenge-specific_reward_asset_id").parentElement.style.visibility = "hidden";
+    });
+</script>
```

Comment:
Don't use js here ? all js should go to xcoin.js file , and you're missing a new empty line at the end of the file